python/paddle/fluid/tests/unittests/test_activation_op.py

```diff
@@ -2973,30 +2973,32 @@ def test_check_grad(self):
 create_test_act_fp16_class(TestSwish, grad_atol=0.85)
 create_test_act_fp16_class(TestHardSwish)
 
-# def create_test_act_bf16_class(parent,
-#                                atol=1e-2,
-#                                grad_check=True,
-#                                grad_atol=0.80):
-#     @unittest.skipIf(not paddle.is_compiled_with_cuda(),
-#                      "core is not compiled with CUDA")
-#     class TestActBF16(parent):
-#         def init_dtype(self):
-#             self.dtype = np.uint16
-
-#         def test_check_output(self):
-#             place = core.CUDAPlace(0)
-#             self.check_output_with_place(place, atol=atol)
-
-#         def test_check_grad(self):
-#             place = core.CUDAPlace(0)
-#             self.check_grad_with_place(
-#                 place, ['X'], 'Out', max_relative_error=grad_atol)
-
-#     cls_name = "{0}_{1}".format(parent.__name__, "bf16")
-#     TestActBF16.__name__ = cls_name
-#     globals()[cls_name] = TestActBF16
-
-# create_test_act_bf16_class(TestRelu)
+
+def create_test_act_bf16_class(parent,
+                               atol=1e-2,
+                               grad_check=True,
+                               grad_atol=0.80):
+    @unittest.skipIf(not paddle.is_compiled_with_cuda(),
+                     "core is not compiled with CUDA")
+    class TestActBF16(parent):
+        def init_dtype(self):
+            self.dtype = np.uint16
+
+        def test_check_output(self):
+            place = core.CUDAPlace(0)
+            self.check_output_with_place(place, atol=atol)
+
+        def test_check_grad(self):
+            place = core.CUDAPlace(0)
+            self.check_grad_with_place(
+                place, ['X'], 'Out', max_relative_error=grad_atol)
+
+    cls_name = "{0}_{1}".format(parent.__name__, "bf16")
+    TestActBF16.__name__ = cls_name
+    globals()[cls_name] = TestActBF16
+
+
+create_test_act_bf16_class(TestRelu)
```

Comment:
@chenwhql @Aganlengzi This bfloat16 test located in test_activation_op.py is failing on the Py3 CI compiled with CUDA. I suspect #37202 caused it to fail. The test is failing only on the Py3 CI. It passes locally with PP compiled with CUDA as well as on the Coverage CI that uses CUDA. It isn't skipped on any of them, as the only skip condition is `@unittest.skipIf(not paddle.is_compiled_with_cuda(), "core is not compiled with CUDA")`, which is not true for PP compiled with GPU. The error shown in the CI log isn't really helpful at all:
![image](https://user-images.githubusercontent.com/89987744/147231742-861887f6-5254-4365-8321-f66f87716d12.png)
Could you please look into this and suggest what has to be changed to fix the failing CI?

Reply:
@piotrekobiIntel did you merge the develop? I checked other PRs and test_activation_op seems good in their Py3 CI.

Reply:
@Aganlengzi I did just now, and the test is still failing. I'd like to fix this CI before changing the location of bfloat16.h, just to be sure that everything else is working correctly. Do you have any suggestions on what can be done or who I should ask for help? As I said above, I believe that #37202 caused the test to fail.

Reply:
@ronny1996 will help check #37202 for UT fail in test_activation_op.py.